### PR TITLE
the home page works properly as a pieces page, with unit tests to pre…

### PR DESCRIPTION
**Pull request by boutell**
…vent regression

### CODE REVIEW

File: test/pieces-pages.js [hunk 1]
```diff
@@ -50,6 +50,15 @@ describe('Pieces Pages', function() {
         }
       },
       afterInit: function(callback) {
+        console.log('>> afterInit');
```
Comment (bobclewell) [inline test/pieces-pages.js]: Do you want to keep this log output around?

File: test/pieces-pages.js [hunk 2]
```diff
@@ -50,6 +50,15 @@ describe('Pieces Pages', function() {
...
+        // In tests this will be the name of the test file,
+        // so override that in order to get apostrophe to
+        // listen normally and not try to run a task. -Tom
+        apos.argv._ = [];
+        return callback(null);
+      },
+      afterListen: function(err) {
```
Comment (bobclewell) [inline test/pieces-pages.js]: Same function with the same name, right below (line 68)?

File: test/pieces-pages.js [hunk 3]
```diff
@@ -162,4 +171,36 @@ describe('Pieces Pages', function() {
...
+    });
+  });
+
+  it('pieces-page as home page: correct permalinks on index page', function(done) {
+    return request('http://localhost:7900/', function(err, response, body) {
+      assert(!err);
+      // Is our status code good?
+      assert.equal(response.statusCode, 200);
+      // Only page one events should show up
+      assert(body.match(/event-001"/));
```
Comment (bobclewell) [inline test/pieces-pages.js]: perhaps a silly question, but what is the unpaired double quote in this line and also in the line below?

File: test/pieces-pages.js (hunk 1 above)
Comment (boutell) [reply test/pieces-pages.js]: Oops, fixed.

File: test/pieces-pages.js (hunk 2 above)
Comment (boutell) [reply test/pieces-pages.js]: Oops, fixed (caused no problems because last one wins, but still).

File: test/pieces-pages.js (hunk 3 above)
Comment (boutell) [reply test/pieces-pages.js]: This one is OK, it is just that I don't care about matching the entire URL here.

**Pull request merged**